lib/API/RelationService.php

```diff
@@ -50,4 +51,42 @@ public function loadFieldRelations(
         array $contentTypeIdentifiers = [],
         ?int $limit = null
     ): array;
+
+    /**
+     * Load single related Location from $fieldDefinitionIdentifier field in $content,
+     * optionally limited by a list of $contentTypeIdentifiers.
+     *
+     * Note: only visible main Location of the related Content will be used.
+     *
+     * @param \Netgen\EzPlatformSiteApi\API\Values\Content $content
+     * @param string $fieldDefinitionIdentifier
+     * @param array $contentTypeIdentifiers
+     *
+     * @return null|\Netgen\EzPlatformSiteApi\API\Values\Location
+     */
+    public function loadLocationFieldRelation(
```

Comment:
Naming nitpick:
I'd prefer this to be named `loadFieldRelationLocation` as we are loading location(s) from the field relation.
Current naming implies that we are loading field relation from the location, which is not true.

I am not sure if I am making sense here :)

Reply:
@iherak Actually, now that you pointed it out, I agree it does make more sense with the naming pattern you proposed. I didn't give it much thought at the first glance.
So the desired format change would be:
loadLocationFieldRelation(s) -> loadFieldRelationLocation(s)
filterLocationFieldRelation(s) -> filterFieldRelationLocation(s)
cc @pspanja

Reply:
Agreed, that's much better 👍

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/190/commits/8a63631cddb0d5b502f1296da922012cd9b45c86